src/ClusterBootstrap/services/repairmanageragent/repairmanageragent.yaml

```diff
@@ -0,0 +1,90 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: repairmanageragent
+  namespace: default
+spec:
+  selector:
+    matchLabels:
+      repairmanageragent-node: pod
+  template:
+    metadata:
+      name: repairmanageragent
+      labels:
+        repairmanageragent-node: pod
+        app: repairmanageragent
+    spec:
+      {% if cnf["dnsPolicy"] %}
+      dnsPolicy: {{cnf["dnsPolicy"]}}
+      {% endif %}
+      hostPID: true
+      nodeSelector:
+        repairmanageragent: active
+      containers:
+      - name: repairmanageragent
+        image: {{cnf["worker-dockerregistry"]}}/{{cnf["dockerprefix"]}}/{{cnf["repairmanageragent"]}}:{{cnf["dockertag"]}}
+        securityContext:
+          capabilities:
+            add: ["SYS_BOOT"]
+        command: ["/run.sh"]
+        imagePullPolicy: Always
+        env:
+        - name: NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        - name: ETCD_SERVER
+          value: "{{cnf['kubernetes_master_node'][0]}}"
+        - name: ETCD_PORT
+          value: "{{cnf['repair-manager']['etcd']['client-port']}}"
```

Comment:
environment variables passed to the docker container allow the repairmanager agent to get and update key values in etcd server.